src/components/App/App.js

```diff
@@ -1,5 +1,43 @@
-import { Frontpage } from 'pages';
+import { Switch, Route, Redirect } from 'react-router-dom';
+import { Frontpage, Dashboard } from 'pages';
+import { useState } from 'react';
 
 export const App = () => {
-    return <Frontpage />;
+    const [user, setUser] = useState(() => {
+        if (localStorage.getItem('user')) {
+            return JSON.parse(localStorage.getItem('user'));
+        } else return [];
+    });
+
+    const userIsAuthenticated = user.length !== 0;
+
+    const authenticateUser = array => {
```

Comment:
When you give down a function into another component you want to call the function itself to be like:

```suggestion
    const authenticateUserHandler = array => {
```

or 

```suggestion
    const handleAuthenticateUser = array => {
```

Same for the logoutUser